### PR TITLE
Improved code style

### PR DESCRIPTION
**Pull request by KrauseFx**
New PR 👍

### CODE REVIEW

**Comment by KrauseFx** [comment]
Taken from https://github.com/fastlane/fastlane/pull/4955 basically

File: .rubocop.yml [hunk 1]
```diff
@@ -98,16 +89,22 @@ Metrics/ParameterLists:
...
+# something = if something_else
+# that's confusing
+Style/ConditionalAssignment:
+  Enabled: false
 
-# Split
-
+# Better to have too much self than missing a self
+Style/RedundantSelf:
+  Enabled: false
```
Comment (neonichu) [inline .rubocop.yml]: 😱

Comment (KrauseFx) [inline .rubocop.yml]:
```
variable = 123
```
that's evil 🙈

Comment (KrauseFx) [inline .rubocop.yml]: 😈

Comment (neonichu) [inline .rubocop.yml]: Ya tu sabes

**Comment by mfurtak** [comment]
👍

**Comment by KrauseFx** [comment]
Great to see this merged, thanks @mfurtak

**Pull request merged**